Xamarin.Forms.Platform.Android/Renderers/ListViewRenderer.cs

```diff
@@ -501,7 +501,8 @@ protected override void OnLayout(bool changed, int l, int t, int r, int b)
 
 			protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
 			{
-				if (_child == null)
+				if (_child == null ||
+				    _child.Element == null)
```

Comment:
```suggestion
```

Reply:
I will make that change as soon as I figure out how :)

I actually noticed a few things in that function that can be cleaned up.